planet/specs.py

```diff
@@ -111,15 +111,18 @@ def get_item_types(product_bundle=None):
     '''If given product bundle, get specific item types supported by Orders
     API. Otherwise, get all item types supported by Orders API.'''
     spec = _get_product_bundle_spec()
-    if product_bundle:
-        item_types = spec['bundles'][product_bundle]['assets'].keys()
-    else:
-        product_bundle = get_product_bundles()
-        all_item_types = []
-        for bundle in product_bundle:
-            all_item_types += [*spec['bundles'][bundle]['assets'].keys()]
-        item_types = set(all_item_types)
-    return item_types
+    try:
+        if product_bundle:
+            item_types = spec['bundles'][product_bundle]['assets'].keys()
```

Comment:
Small thing, but let's not put any empty lines after `try`, `if`, `for`, `while`, etc. I think an empty line *before* those can help readability, but not after.

Reply:
Also it looks like `item_types` is never returned in this case?

Reply:
I would suggest this for line 116: `return set(spec['bundles'][product_bundle]['assets'].keys())`

Reply:
> Small thing, but let's not put any empty lines after `try`, `if`, `for`, `while`, etc. I think an empty line _before_ those can help readability, but not after.

You got it.



> Also it looks like `item_types` is never returned in this case?

`item_types` is returned on line 123, after the `if`/`else` statement.



> I would suggest this for line 116: `return set(spec['bundles'][product_bundle]['assets'].keys())`

On it!

Reply:
For readability, I'll move line 123 (`return item_types`) to after `except KeyError`.